test/flask/snaps.spec.ts

```diff
@@ -105,12 +108,22 @@ describe("snaps", function () {
       expect(await invokeAction).to.equal(false);
     });
 
-    it("should invoke IN APP NOTIFICATIONS and check for a text", async function (this: TestContext) {
+    it("should invoke IN APP NOTIFICATIONS", async function (this: TestContext) {
+      const notifications = await metamask.snaps.notificationObserver();
+
+      await metamask.snaps.invokeSnap(testPage, snapId, "notify_inApp");
+
+      await metamask.snaps.waitForAmountOfNotifications(1);
+
+      // Metamask doesn't allow to invoke more than one notification right away
+      // the time between showing next notification should be minimum 5 seconds
+      await metamask.page.waitForTimeout(5000);
```

Comment:
let's not do this, takes a long time as it is. Either run invoke notification from some other snap or don't test this